src/addons/MountNode/lib/handleClassNamesChange.js

```diff
@@ -0,0 +1,18 @@
+import _ from 'lodash'
+
+import computeClassNames from './computeClassNames'
+import computeClassNamesDifference from './computeClassNamesDifference'
+
+const prevClassNames = new Map()
```

Comment:
Yes, it's a planned solution. We use a DOM node as the key in Map, so we will have the same classes for the same node.

Here is a simple example:
```jsx
const div = document.createElement('div')
const Tree = () => (
  <>
   <MountNode className='foo' node={div} />
   <MountNode className='bar' node={div} />
  </>
)
```

Because we have the same Map, we will have `foo bar` className on the `div` element after `Tree` will be mounted.